eng/emitter-package.json

```diff
@@ -1,6 +1,13 @@
 {
```

Comment:
@mikeharder we are adding the emitter-package-lock.json files but we haven't updated our generate scripts (https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/TypeSpec-Project-Generate.ps1) to use this yet so I assume there is a workitem for that as well.

Reply:
Yes, we will update the common scripts after all lockfiles are added to all repos.

Reply:
Tracking issue: https://github.com/Azure/azure-sdk-tools/issues/6349